Application/EdFi.Ods.AdminApi/Infrastructure/AdminApiEndpointBuilder.cs

```diff
@@ -44,8 +44,8 @@ public static AdminApiEndpointBuilder MapDelete(IEndpointRouteBuilder endpoints,
     public void BuildForVersions(params AdminApiVersions.AdminApiVersion[] versions)
     {
         if (versions.Length == 0) throw new ArgumentException("Must register for at least 1 version");
-        if (_route == null) throw new Exception("Invalid endpoint registration. Route must be specified");
-        if (_handler == null) throw new Exception("Invalid endpoint registration. Handler must be specified");
+        if (_route == null) throw new ArgumentException("Invalid endpoint registration. Route must be specified");
+        if (_handler == null) throw new ArgumentException("Invalid endpoint registration. Handler must be specified");
```

Comment:
`_route` and `_handler`  are not arguments to this function, therefore `ArgumentException` is inappropriate. Probably `InvalidOperationException` is appropriate.